### PR TITLE
Feat: [OV-41715] icon button ghost secondary

### PR DESCRIPTION
**Pull request by vicky-comeau**

### CODE REVIEW

File: packages/Button/src/button.scss [hunk 1]
```diff
@@ -12,7 +12,7 @@
   --ids-btn-padding: #{tokens.$space-3};
   --ids-btn-padding-mobile: 1.2rem;
   --ids-btn-border-radius: #{tokens.$border-radius-sm};
-  --ids-btn-icon-color: currencolor;
+  --ids-btn-icon-color: currentColor;
   --ids-btn-icon-margin: #{tokens.$space-1};
   --ids-btn-z-index: #{tokens.$z-index-10};
   --ids-btn-focus-position: -0.3rem;
```
Comment (ofrogon) [inline packages/Button/src/button.scss:18]: `--ids-btn-z-index`, `--ids-btn-focus-position` and `--ids-btn-shadow` are not used anymore and can be deleted

File: packages/Button/src/button.scss [hunk 2]
```diff
@@ -74,6 +74,13 @@
   --ids-loader-border-ghost-danger: 2px solid rgb(249 97 85 / 30%);
   --ids-loader-ghost-danger: #{tokens.$coral-500};
 
+  /* Ghost, variant: Secondary */
+  --ids-btn-text-ghost-secondary: #{tokens.$grey-600};
+  --ids-btn-text-ghost-secondary-hover: #{tokens.$grey-800};
+  --ids-btn-text-ghost-secondary-disabled: #{tokens.$grey-400};
+  --ids-loader-border-ghost-secondary: 2px solid rgb(96 107 121 / 30%);
+  --ids-loader-ghost-secondary: #{tokens.$grey-600};
+
```
Comment (ofrogon) [inline packages/Button/src/button.scss:83]: `--ids-btn-text-secondary-hover` variable setup is missing

Comment (ofrogon) [reply packages/Button/src/button.scss:83]: Also `--ids-btn-border-secondary-disabled`

Comment (ofrogon) [reply packages/Button/src/button.scss:83]: `--ids-btn-text-danger` is never used

**Review by franckgaudin — APPROVED** [review]

**Review by ofrogon — APPROVED** [review]

**Pull request merged**